src/app/applyContextMenu.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @author Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { clipboard, Menu, MenuItem } from 'electron'
+import { isExternalLink } from './externalLinkHandlers.js'
+
+/**
+ * Enable context menu by right click
+ *
+ * @param {import('electron').BrowserWindow} browserWindow - Browser window
+ */
+export function applyContextMenu(browserWindow) {
```

Comment:
This is the only module in the nodejs process with ES Modules. It seems to work as the electron-forge webpack config transpiles ES Modules to Commonjs, but it is inconsistent with all the other modules in the main process and doesn't follow the package.json type.